default/stringtables/en.txt

```diff
@@ -4071,6 +4071,20 @@ TROOP_TEXT
 
 The Troop meter on a planet represents the power of the defending ground forces. The invading force must send a more powerful force (i.e. a higher cumulative number) to conquer a planet.  Ships must be built with special parts for carrying troops, and such troops cannot invade if the defending planet has active [[encyclopedia SHIELDS_TITLE]].'''
 
+ENVIRONMENT_TITLE
+Environment
+
+ENVIRONMENT_TEXT
+'''Eleven different environments can be encountered when exploring the planetary systems on the galactic map: Inferno, Radiated, Toxic, Barren, Desert, Tundra, Swamp, Terran, Ocean, Asteroids, and Gas Giant.
```

Comment:
> May be avoid to mention exact number of planet types, jsut to change less if new one will come?

Normally, I'd agree completely, but I don't see any plans for new environments and, unless there's a *really* good reason, would argue for there to be a Special instead, we don't have a strong enough creature mix for what we've got.  

I would suggest splitting it into 9 standard habitable planets and two special types that are normally not colonised but have different in game effects.

I agree on the keys though.